### PR TITLE
Add topics for Galaxy and Beacon.  Update QC topic.

### PR DESCRIPTION
**Pull request by deflaux**
* Fixes https://github.com/googlegenomics/readthedocs/issues/102 @bryeung and @mfiume please take a look at [Beacon](http://googlegenomics.readthedocs.org/en/staging/use_cases/browse_genomic_data/beacon.html)

* Fixes https://github.com/googlegenomics/readthedocs/issues/103 @kellrott please take a look at [Galaxy](http://googlegenomics.readthedocs.org/en/staging/use_cases/run_familiar_tools/galaxy.html) 

* Fixes https://github.com/googlegenomics/readthedocs/issues/104 @cuiping and @gregmcinnes please take a look at [QC](http://googlegenomics.readthedocs.org/en/staging/use_cases/perform_quality_control_checks/qc_codelab.html)

Please feel free to add more to these topics and edit/correct.  Just click on "Edit on GitHub" in the upper right hand corner of the page on readthedocs and send a pull request, thanks!

### CODE REVIEW

File: docs/source/use_cases/compress_or_decompress_many_files/index.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
    <br />
 
 =================================================
-Compress/Decompress files in Google Cloud Storage
+Compress/Decompress files in Cloud Storage
 =================================================
```
Comment (mbookman) [inline docs/source/use_cases/compress_or_decompress_many_files/index.rst]: Please shorten the "underline" to match the text.

File: docs/source/use_cases/discover_public_data/isb_cgc_data.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ __ RenderedVersion_
...
+Use the power of BigQuery to analyze the wealth of data created by `The Cancer Genome Atlas <http://cancergenome.nih.gov/>`_ (TCGA) project! The Institute for Systems Biology (ISB) has created and made public a dataset based on the open-access TCGA data including somatic mutation calls, clinical data, mRNA and miRNA expression, DNA methylation and protein expression from 33 different tumor types. It's part of their `Cancer Genomics Cloud`_, funded by the National Cancer Institute. They've also created public github repositories with `examples in R <https://github.com/isb-cgc/examples-R>`_ and `examples in Python <https://github.com/isb-cgc/examples-Python>`_  so you can try out sample queries and analyses in `Google Cloud Datalab <https://cloud.google.com/datalab/>`_.
```
Comment (mbookman) [inline docs/source/use_cases/discover_public_data/isb_cgc_data.rst]: Add the datalab reference conf.py?

File: docs/source/use_cases/run_familiar_tools/bioconductor.rst [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Run Bioconductor on Google Compute Engine
+Run Bioconductor on Compute Engine
 =========================================
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/bioconductor.rst]: Update the "===" line to match the title

File: docs/source/use_cases/run_familiar_tools/datalab.rst [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Run iPython Notebooks on Google Compute Engine
+Run iPython Notebooks on Compute Engine
 ===============================================
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/datalab.rst]: Update the "===" line to match the title

File: docs/source/use_cases/run_familiar_tools/galaxy.rst [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Run Galaxy on Compute Engine
+===================================
```
Comment (mbookman) [inline docs/source/use_cases/run_familiar_tools/galaxy.rst]: Update the "===" line to match the title

**Comment by mbookman** [comment]
Just a few nits, but otherwise good. Please take a look through the overall diff for all the places that the page titles have changed and match up the length of the "====". I know it is not a strict technical requirement, but nice to keep consistent.

**Comment by mfiume** [comment]
Beacon section looks great, thanks @deflaux.

**Pull request merged**